src/Footer/footer.style.js

```diff
@@ -1,53 +1,11 @@
 import styled from "styled-components";
 
 const FooterStyles = styled.div`
-  width: 100%;
-
-  .footer {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    text-align: center;
-    height: 86px;
-    flex-wrap: wrap;
-    border-top: solid 0.5px #cdcdcd;
-    margin: 0 15%;
-  }
-
-  .link {
-    display: flex;
-    border-right: 1px solid #cdcdcd;
-    padding: 0 16px;
+  a {
+    display: block;
```

Comment:
are you sure this doesn't do weird things?

Reply:
margin wont work otherwise.
I tested it.